tests/unit/NextStepUtilsTest.ts

```diff
@@ -553,13 +571,5 @@ describe('libs/NextStepUtils', () => {
                 expect(result).toMatchObject(optimisticNextStep);
             });
         });
-
-        describe('it generates a nullable optimistic nextStep', () => {
-            test('closed status', () => {
-                const result = NextStepUtils.buildNextStep(report, CONST.REPORT.STATUS_NUM.CLOSED);
-
-                expect(result).toBeNull();
-            });
-        });
```

Comment:
I believe this usecase is only in the Cancel payment flow and even in that one I think its fair so show the Finished no further action required message. So its safe to remove